packages/pretty-format/src/__tests__/__snapshots__/react.test.js.snap

```diff
@@ -1,43 +1,43 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`ReactElement plugin highlights syntax 1`] = `
-"<Mouse</>
+"<cyan><Mouse</>
```

Comment:
Why is cyan added here?

Reply:
This is side effect of supporting cyan and background colors in `ConvertAnsi` plugin. I can revert that change and that diff produces the correct colors in a more specific way.

Reply:
With all your explanation, I'm still not sure why the cyan is printed here at all. Is this just syntax highlighting for the React component itself? If yes, then this change is fine. If no, I'd like to understand better why the cyan is there now.

Reply:
Aha, got it. What you said. When we changed from raw ANSI codes to `ConvertAnsi` plugin, we overlooked that not all colors in the default React plugin color options were supported, so there was a loss of information. The change to display cyan as the default color for markers is correct, but really an unintended side effect.

Reply:
Yup, makes sense to me now. Thanks for explaining.